src/BaseCharts.js

```diff
@@ -88,18 +118,60 @@ export function generateChart(chartId, chartType) {
   }
 }
 
-export const Bar = generateChart('bar-chart', 'bar')
-export const HorizontalBar = generateChart(
+export const Bar = /* #__PURE__ */ generateChart(
+  'bar-chart',
+  'bar',
+  BarController
+)
+
+export const HorizontalBar = /* #__PURE__ */ generateChart(
```

Comment:
Let's be in sync with chart.js and drop it.

Reply:
dropped